package.json

```diff
@@ -15,6 +15,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "testjest": "jest",
```

Comment:
Unfortunately, your script works with errors. 
![2023-04-01_08h49_40](https://user-images.githubusercontent.com/47754466/229271131-85c712d0-9e94-4b3c-9032-d545c43e4d3d.png)

Let's add some config files for Jest. (babel.config.js, jest.config.js)